src/components/tests/SourcesTree.spec.js

```diff
@@ -85,4 +86,622 @@ describe("SourcesTree", () => {
+      const node = component
+        .instance()
+        .renderItem(item, 1, false, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show icon for folder with arrow", async () => {
+      const item = createMockItem("folder", "folder", []);
+      const node = component
+        .instance()
+        .renderItem(item, 1, false, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show icon for folder with expanded arrow", async () => {
+      const item = createMockItem("folder", "folder", []);
+      const node = component
+        .instance()
+        .renderItem(item, 1, false, null, true, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show focused item for folder with expanded arrow", async () => {
+      const item = createMockItem("folder", "folder", []);
+      const node = component
+        .instance()
+        .renderItem(item, 1, true, null, true, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show source item with source icon", async () => {
+      const node = component
+        .instance()
+        .renderItem(itemOne, 1, false, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show source item with source icon with focus", async () => {
+      const node = component
+        .instance()
+        .renderItem(itemOne, 1, true, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show domain item", async () => {
+      const item = createMockItem("root", "root");
+      const node = component
+        .instance()
+        .renderItem(item, 0, false, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show domain item as debuggee", async () => {
+      const item = createMockItem("root", "http://mdn.com");
+      const node = component
+        .instance()
+        .renderItem(item, 0, false, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it("should show domain item as debuggee with focus and arrow", async () => {
+      const item = createMockItem("root", "http://mdn.com", []);
+      const node = component
+        .instance()
+        .renderItem(item, 0, true, null, false, { setExpanded: jest.fn() });
+      expect(node).toMatchSnapshot();
+    });
+
+    it(
+      "should not show domain item if projectRoot is not empty " +
+        "even if depth is 0",
+      async () => {
+        const componentWithRoot = render({
+          projectRoot: "project-root"
+        }).component;
+
+        const node = componentWithRoot
+          .instance()
+          .renderItem(itemOne, 0, false, null, false, {
+            setExpanded: jest.fn()
+          });
+        expect(node).toMatchSnapshot();
```

Comment:
it might be nice to replace some of the match snapshots with element tests like: exists or has classes:
https://github.com/airbnb/enzyme/blob/master/docs/api/selector.md